### PR TITLE
Nominator: Be more verbose when we are not nominating

### PR DESCRIPTION
**Pull request by Geod24**
This should help better debug the reason(s) why a node is not nominating.
Currently we only see a bunch of 'Nominating X at Y" but nodes can get stuck.

### CODE REVIEW

File: source/agora/consensus/protocol/Nominator.d [hunk 1]
```diff
@@ -314,12 +314,15 @@ extern(D):
     {
         this.ledger.prepareNominatingSet(data, MaxTransactionsPerBlock, this.nomination_start_time);
         if (data.tx_set.length < 1)
-            return false;  // not ready to nominate yet
+        {
+            this.log.trace("checkNominate(): No transaction to nominate yet");
```
Comment (hewison-chris) [inline source/agora/consensus/protocol/Nominator.d]:
```suggestion
            this.log.trace("prepareNominatingSet(): No transaction to nominate yet");
```

**Review by hewison-chris — CHANGES_REQUESTED** [review]
LGTM just a tiny change requested

**Review by linked0 — APPROVED** [review]
LGTM

File: source/agora/consensus/protocol/Nominator.d (hunk 1 above)
Comment (Geod24) [reply source/agora/consensus/protocol/Nominator.d]: Good catch!

**Comment by Geod24** [comment]
Fixed method name

**Review by hewison-chris — APPROVED** [review]
LGTM

**Review by kchulj — APPROVED** [review]
LGTM

**Pull request merged**